packages/blockchain-wallet-v4-frontend/src/modals/CoinifyExchangeData/ISignThis/index.js

```diff
@@ -16,17 +18,37 @@ const ButtonContainer = styled.div`
   margin-left: 5%;
   width: 20%;
 `
+const TimerContainer = styled.div`
+  width: 66%;
+  padding-bottom: 5px;
+  display: flex;
+  justify-content: flex-end;
+`
+const QuoteExpiredText = styled(Text)`
+  span {
+    margin-right: 5px;
+  }
+  span:first-of-type {
+    font-style: italic;
+  }
+`
 
 const helpers = [
   {
     question: <FormattedMessage id='scenes.buysell.coinify.isx.question1' defaultMessage='Why do I have to do this?' />,
     answer: <FormattedMessage id='scenes.buysell.coinify.isx.answer1' defaultMessage="Completing the identity verification process allows you to buy and sell at higher limits. If you don't submit this information, you will only be able to buy and sell up to €300." />
   }
 ]
-
+const getExpiredBtcValues = (q) => q.quoteCurrency === 'BTC' ? `${q.quoteAmount / 1e8}` : `${q.baseAmount / 1e8}`
```

Comment:
We're doing this kind of things in several places. We should probably use functions for that if we don't have them somewhere already